docs-src/stargate-develop/modules/quickstart/pages/qs-rest.adoc

```diff
@@ -50,5 +50,5 @@ include::develop:page$api-rest/rest-deleting-rows.adoc[Deleting rows in your tab
 
 Voila! For more information on the REST API, see the see the
 xref:develop:dev-with-rest.adoc[Using the REST API]
-or the https://stargate.io/docs/stargate/1.0/attachments/restv2.html[REST API] in the
+or the https://stargate.io/docs/latest/develop/attachments/restv2.html[REST API] in the
```

Comment:
@polandll how about making it like `xref:develop:api.adoc[REST API]` so we make folks to the API landing page where they come to know about other APIs too? Thoughts?